src/style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  max-width: 1200px;
+  min-width: 1024px;
```

Comment:
bad idea, remove min-width. the horizontal scroll appears on your page on small screens.